controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -78,7 +79,6 @@ type reconcileInfo struct {
 	request           ctrl.Request
 	bmcCredsSecret    *corev1.Secret
 	events            []corev1.Event
-	errorMessage      string
```

Comment:
Why does it need to be removed? It looks like this change has nothing to do with the current commit, I think it's better to make it a separate commit or a separate PR to avoid misunderstandings.

Reply:
I removed it because it was unused code. Sure, I can leave it as it is